kotlinx-coroutines-debug/test/StartModeProbesTest.kt

```diff
@@ -140,6 +140,7 @@ class StartModeProbesTest : DebugTestBase() {
     fun testLazy() = runTest({ it is CancellationException }) {
         launch(start = CoroutineStart.LAZY) {  }
         actor<Int>(start = CoroutineStart.LAZY) {  }
+        @Suppress("DEPRECATION_ERROR")
```

Comment:
We can probably gradually start removing it from irrelevant tests (i.e. blockhound and debug probes)